packages/@okta/vuepress-site/docs/guides/quickstart/using-console/index.md

```diff
@@ -2,7 +2,7 @@
 title: Using the Console
 ---
 
-The Developer Console is the web UI that provides you with a window into your org and lets you configure and manage it. The URL to access your Developer Console consists of your Okta domain plus `-admin`, e.g., `https://dev-l33337.okta.com`.
+The Developer Console is the web UI that provides you with a window into your org and lets you configure and manage it. The URL to access your Developer Console consists of your Okta domain plus `-admin`, e.g., `https://dev-l33337-admin.okta.com`.
```

Comment:
Avoid latin abbreviations - use "for example" instead: 

The Developer Console is the web UI that provides you with a window into your org and lets you configure and manage it. The URL to access your Developer Console consists of your Okta domain plus `-admin`, for example, `https://dev-l33337-admin.okta.com`.